internal/app/wwctl/node/list/main.go

```diff
@@ -44,6 +44,7 @@ func CobraRunE(vars *variables) func(cmd *cobra.Command, args []string) (err err
 				for _, val := range nodeInfo.Output[1:] {
 					ph.Append(strings.Split(val, ":=:"))
 				}
+				ph.SetColMinWidth(3, 10) // set the min width of last column (value column) in order to avoid of empty newlines.
```

Comment:
Rather than changing this here, which only affects `wwctl node list`, it should be applied in `NewPrintHelper` so that it affects all of `wwctl`. It also needs to remove all trailing whitespace; this appears to have removed some, but the lines still have trailing whitespace.

I'm uncertain whether tablewriter is really the best fit for this; but the quickest thing might be to have tablewriter [render to a string](https://github.com/olekukonko/tablewriter?tab=readme-ov-file#render-table-into-a-string) and then use a regular expression to remove `(?m) *$` before printing.

Reply:
yeah, thanks, I've reflected the code as suggested and adjusted test cases as well.